client/src/test/java/org/eclipse/hono/client/impl/HonoClientImplTest.java

```diff
@@ -59,7 +59,7 @@
      * Global timeout for each test case.
      */
     @Rule
-    public Timeout timeout = Timeout.seconds(5);
+    public Timeout timeout = new Timeout(2, TimeUnit.SECONDS);
```

Comment:
is there a reason why you changed this?

Reply:
no. this format works for me if i want to change the timeunit (during debugging). forgot to change it back afterwards